VERSION.cmake

```diff
@@ -1,11 +1,11 @@
 SET(VERSION_MAJOR "2")
 SET(VERSION_MINOR "50")
-SET(VERSION_PATCH "0")
+SET(VERSION_PATCH "1")
 SET( VERSION "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}${GIT_SHA1_VERSION}" )
 
 ##### This is needed for the libyui-qt core ONLY.
 ##### These will be overridden from exports in LibyuiConfig.cmake
-SET( SONAME_MAJOR "10" )
-SET( SONAME_MINOR "0" )
+SET( SONAME_MAJOR "5" )
+SET( SONAME_MINOR "1" )
```

Comment:
This was changed in the previous PR, probably restored in git merge. Please keep so version 10.

Reply:
I had to do this to even get a local `rake osc:build` to work. But since that libyui version is now merged, I'll revert that last commit.